generate_queries_openai.py

```diff
@@ -104,8 +96,9 @@ def parse_bad_good(data):
 
                 if args.good_bad:
                     question, bad_question = parse_good_bad(output)
-                elif args.bad_good:
-                    question, bad_question = parse_bad_good(output)
+                    print('question', question)
+                    print('badquestion', bad_question)
```

Comment:
This may result on a lot of output to the terminal, what's the reason behind it?

Reply:
my bad, sorry. fixed!